backend/src/remoteitInstaller.ts

```diff
@@ -1,9 +1,23 @@
 import Installer from './Installer'
-import { CLI_VERSION } from './constants'
+import {
+  BASE_URL,
+  CLI_URL,
+  CLI_VERSION,
+  CONNECTD_URL,
+  CONNECTD_VERSION,
+  DEMUXER_VERSION,
+  MUXER_VERSION,
+} from './constants'
```

Comment:
This is the right file to make the array of installers - where each one represents a binary to install... [remoteit, connectd, muxer, demuxer]...